easybuild/easyconfigs/f/fullrmc/fullrmc-3.2.0-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'Bundle'
+
+name = 'fullrmc'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://bachiraoun.github.io/fullrmc'
+description = """Reverse Monte Carlo (RMC) is probably best known for its applications
+ in condensed matter physics and solid state chemistry. fullrmc which stands for
+ FUndamental Library Language for Reverse Monte Carlo is different than traditional
+ RMC but a stochastic modelling method to solve an inverse problem whereby
+ an atomic/molecular model is adjusted until its atoms position havei
+ the greatest consistency with a set of experimental data."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('VMD', '1.9.3', versionsuffix),
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
```

Comment:
@hajgato Please also add `exts_download_dep_fail = True` to make sure all required extensions are listed, see https://github.com/easybuilders/easybuild-easyblocks/pull/1377

Reply:
@boegel Why do not we make it default?

Reply:
@boegel: Or at least issue a warning by default if something is downloaded.

Reply:
Making it default would break backward compatibility, and there are quite a bit of older easyconfig that would definitely need work. We hope to make it default in EasyBuild 4.x

We can consider the warning though...